pkg/client/pubsub.go

```diff
@@ -50,21 +50,40 @@ func (t Topic) Publish(ctx context.Context, msg []byte) error {
 	return pubsub.Topic{Client: t.Client}.Publish(ctx, msg)
 }
 
-func (t Topic) Subscribe(ctx context.Context) (Subscription, error) {
-	out := make(chan []byte, 32)
+func (t Topic) Subscribe(ctx context.Context, opts ...SubOpt) (Subscription, error) {
+	sub := Subscription{
+		name: t.name,
+		c:    make(chan []byte, 32),
+	}
 
-	release, err := pubsub.Topic{Client: t.Client}.Subscribe(ctx, out)
-	return Subscription{
-		name:   t.name,
-		cancel: release,
-		c:      out,
-	}, err
+	for _, opt := range opts {
```

Comment:
nit:  we've been calling the function `option` in other places, so let's keep it consistent.

Reply:
@lthibault Do you refer to `SubOpt`, `opt` or `opts`. Which should be renamed to option?

Reply:
ah sorry, i meant we should replace `opt` with `option`.  The rest is good to go.

Reply:
So `for _, option := range opt { ... }`